src/components/grapher/Grapher.vue

```diff
@@ -3,12 +3,22 @@
     <svg class="grapher-svg"><g> <!-- Note: we wrap the entire svg in a g so that svgPanZoom doesn't
             have to create a new g and Vue loses track of the DOM elements. -->
       <!-- Note:Inside svg, 1px = 1 eight-to-five step -->
-      <g class="grapher--line-container">
+      <g
+        v-if="fieldRect"
```

Comment:
When disabling the field rectangle, it still removes the green fill. How about we create a new issue, and figure this out in a separate PR?